### DISCUSSION TITLE
Is there any way admin can get to know how many user is been part of the team.

### DISCUSSION THREAD

**Discussion by 007divyachawla** [juice-shop/multi-juicer#233] [General]
Is there any way admin can get to know how many user is been part of the team.

**Comment by J12934** [comment]
Hi @007divyachawla 

No we do not track / have no way to tell how many users are part of a team.

**Comment by 007divyachawla** [comment]
Hello @J12934 can we track the same from the Grafana Dashboard?

**Comment by J12934** [comment]
No, sorry
